docs/source/concepts/ledger-model/ledger-structure.rst

```diff
@@ -185,6 +189,12 @@ below shows all its proper subtransactions on the right (yellow boxes).
    :align: center
    :width: 100%
 
+To illustrate contract keys, suppose that the contract key for a `PaintOffer` consists of the reference number and the painter as the key maintainer.
```

Comment:
It's been a while since this was first mentioned, so I think it's worth having the link to what these are again.
```suggestion
To illustrate :ref:`contract keys <daml-ref-contract-keys>`, suppose that the contract key for a `PaintOffer` consists of the reference number and the painter as the key maintainer.
```

Reply:
The concept of a maintainer hasn't been introduced yet, and I don't think you need it at this point.

Reply:
Added the link.

Reply:
Moved the maintainers to the integrity section, but kept the notation with `@` and `&`.